[2025-08-11T07:27:01.647+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_pipeline.model_evaluation manual__2025-08-11T07:25:58.461541+00:00 [queued]>
[2025-08-11T07:27:01.665+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_pipeline.model_evaluation manual__2025-08-11T07:25:58.461541+00:00 [queued]>
[2025-08-11T07:27:01.666+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-08-11T07:27:01.701+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): model_evaluation> on 2025-08-11 07:25:58.461541+00:00
[2025-08-11T07:27:01.713+0000] {standard_task_runner.py:60} INFO - Started process 1303 to run task
[2025-08-11T07:27:01.721+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ml_pipeline', 'model_evaluation', 'manual__2025-08-11T07:25:58.461541+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/ml_pipeline.py', '--cfg-path', '/tmp/tmplj5x84f3']
[2025-08-11T07:27:01.728+0000] {standard_task_runner.py:88} INFO - Job 93: Subtask model_evaluation
[2025-08-11T07:27:01.859+0000] {task_command.py:423} INFO - Running <TaskInstance: ml_pipeline.model_evaluation manual__2025-08-11T07:25:58.461541+00:00 [running]> on host bd0e889be214
[2025-08-11T07:27:02.026+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarah' AIRFLOW_CTX_DAG_ID='ml_pipeline' AIRFLOW_CTX_TASK_ID='model_evaluation' AIRFLOW_CTX_EXECUTION_DATE='2025-08-11T07:25:58.461541+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-11T07:25:58.461541+00:00'
[2025-08-11T07:27:08.211+0000] {logging_mixin.py:188} INFO - STDOUT:
[2025-08-11T07:27:08.212+0000] {logging_mixin.py:188} INFO -  üèÉ View run loud-dog-404 at: https://dagshub.com/SamMebarek/mlops-som.mlflow/#/experiments/0/runs/eadeb8ce72024c59a3381465527a68a3
üß™ View experiment at: https://dagshub.com/SamMebarek/mlops-som.mlflow/#/experiments/0
Evaluation completed. R2: 0.9998, MAE: 1.1356
[2025-08-11T07:27:08.213+0000] {logging_mixin.py:188} INFO - STDERR:
[2025-08-11T07:27:08.214+0000] {logging_mixin.py:188} INFO -  2025-08-11 07:27:05,594 - evaluation.utils.common - INFO - YAML charg√© : /opt/***/project/config/config.yaml
2025-08-11 07:27:05,603 - evaluation.utils.common - INFO - YAML charg√© : /opt/***/project/config/params.yaml
2025-08-11 07:27:05,608 - evaluation.utils.common - INFO - R√©pertoire cr√©√© ou existant : /opt/***/project/evaluation
2025-08-11 07:27:06,311 - __main__ - INFO - Loaded processed data: /opt/***/project/data/processed/clean_data.csv (shape=(108060, 14))
2025-08-11 07:27:06,356 - __main__ - INFO - Loaded model: /opt/***/project/models/xgb_model.pkl
2025-08-11 07:27:06,426 - __main__ - INFO - Computed metrics: {'r2_score': 0.9997932180824073, 'mae': 1.135571365098727}
2025-08-11 07:27:06,427 - evaluation.utils.common - INFO - R√©pertoire cr√©√© ou existant : /opt/***/project/evaluation
2025-08-11 07:27:06,431 - evaluation.utils.common - INFO - JSON sauvegard√© : /opt/***/project/evaluation/scores.json
2025-08-11 07:27:06,431 - __main__ - INFO - Metrics saved to: /opt/***/project/evaluation/scores.json
2025/08/11 07:27:06 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh(<full-path-to-git-executable>)

All git commands will error until this is rectified.

This initial message can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|silent|none|n|0: for no message or exception
    - warn|w|warning|log|l|1: for a warning message (logging level CRITICAL, displayed by default)
    - error|e|exception|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet

2025-08-11 07:27:07,450 - __main__ - INFO - Metrics logged to MLflow.
[2025-08-11T07:27:08.215+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-08-11T07:27:08.230+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=ml_pipeline, task_id=model_evaluation, execution_date=20250811T072558, start_date=20250811T072701, end_date=20250811T072708
[2025-08-11T07:27:08.276+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-11T07:27:08.315+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
